src/lib/isSemVer.js

```diff
@@ -9,9 +9,9 @@ import multilineRegexp from './util/multilineRegex';
  */
 const semanticVersioningRegex = multilineRegexp([
   '^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)',
-  '(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))',
-  '?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$',
-]);
+  '(?:-((?:0|[1-9]\\d*|\\d*[a-z-][0-9a-z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-z-][0-9a-z-]*))*))',
```

Comment:
Can explain these changes too?

Reply:
It was the only way to test/exec ```multilineRegexp``` second parameter. The method is only used in this function and there is no difference with the old regex except the fact that this one is shorter.
If we want 100% coverage we have to choose between keeping this little hack or removing completely flags parameter from ```multilineRegexp``` (We may use this function in other validators in the future)